_posts/2021-05-28-seqan-3-0-3-release.md

```diff
@@ -0,0 +1,133 @@
+  Contains alphabet related entities, like `seqan3::dna4`.
+* `#include <seqan3/alignment/*.hpp>`:
+  Contains sequence alignment related entities, like `seqan3::align_pairwise`.
+* `#include <seqan3/io/*.hpp>`:
+  Contains I/O related entities, like `seqan3::sequence_file_input`.
+* `#include <seqan3/search/*.hpp>`:
+  Contains search related entities, like `seqan3::search`.
+
+We also moved containers and views from the former range module into related Sequence Analysis Module (e.g., `seqan3::bitpacked_sequence` and `seqan3::translate` are now part of the alphabet module).
+
+All Sequence Analysis Modules will have strong [API Stability](https://docs.seqan.de/seqan/3.0.3/about_api.html#api_stability) guarantees. So far we finished declaring the stability in our documentation for alphabets, containers, and ranges. Sequence Alignment and I/O will follow in the next release.
+
+#### General Purpose Modules
+
+* `#include <seqan3/argument_parser/*.hpp>`:
+  Our argument parser library that helps to write apps.
+* `#include <seqan3/range/*.hpp>` (split since 3.0.3):
+  We moved most entities within this module into the other modules: alphabet, alignment, io, search or utility. But, we also deprecated some of them that did not fit our scope anymore.
+* `#include <seqan3/core/*.hpp>`:
+  Our internal module for entities that are shared across our sequence analysis modules and are needed to implement our own algorithms and data structures.
+* `#include <seqan3/utility/*.hpp>`:
+  utility contains entities that are completely unrelated to biology-problems. We think of these as something that could be made separate libraries.
+
+All General Purpose Modules will have no [API Stability](https://docs.seqan.de/seqan/3.0.3/about_api.html#api_stability) guarantees.
+
+### :lock: API Stability
+
+We are especially thrilled to announce 3.0.3, because this release should be the first one that _just compiles_<sup>:tm:</sup> your app when upgrading from 3.0.2 to 3.0.3. Arguably, you will encounter scattered "deprecation" notices, but all of those messages should point you to an upgrade path. Unless you treat warnings as errors, your app will still compile even when encountering deprecation notices.
+
+Furthermore, we see this release as a test run of our way to handle API Stability in the upcoming era of stable releases. We are happy to hear your feedback, so please let us know whether it worked for you and if something could be improved. We believe we found a good system for hinting changes between releases to our users.
+
+But how do we know that we did not miss anything? Our idea is simple: compile our current SeqAn version (3.0.3 in this case) against the tests of our previous Release (3.0.2). If everything compiles and the previous tests pass, we most likely did not break our API.
+
+We counted just two real breaking API changes that can be found in our [changelog](https://docs.seqan.de/seqan/3.0.3/about_changelog.html).
+
+### PacBio Phred Score alphabet
+
```

Comment:
This needs another header

e.g. `### Highlights` (?)
And then the examples should be `####`

Reply:
Last time we just put the highlights at the same level as everything else https://www.seqan.de/release/2020/10/13/seqan-3-0-2-release.html

Reply:
### Highlights

would work to

Reply:
When I read it the first time, I was kinda confused because it seemed to belong to API-Stability. On a second read, it seems fine :)

Reply:
I.e., it's good as is